### ISSUE TITLE
Pdf2pic typings and readme exemple

### ISSUE THREAD

**Issue by hadrien-f** [yakovmeister/pdf2image#93]
Hello,
I use Pdf2pic and just upgraded the package but there is one thing I don't understand :

So while I use to do :
```
  pdf2pic.convertToBase64(filename).then( img => ...
```
Now if I understand it has changed to : 

```
fromPath(filePath, options).convert(1, true).then( img => ...
```
But I get the following typescript error : 
```
Property 'convert' does not exist on type 'Convert'
```
So according to the typing file I should do : 
```
fromPath(filePath, options)(1, true).then( img => ...
```

Can you confirm this ? 

Thanks for the work by the way.

**Comment by superjose** [comment]
Same question here!!!

**Comment by superjose** [comment]
@hadrien-f 
Going through the docs again, I think the Typings are the correct, and the documentation is a little bit outdated. 

If you look at the first part of the docs, 
```
import { fromPath } from "pdf2pic";

const options = {
  density: 100,
  saveFilename: "untitled",
  savePath: "./images",
  format: "png",
  width: 600,
  height: 600
};
const storeAsImage = fromPath("/path/to/pdf/sample.pdf", options);
const pageToConvertAsImage = 1;

storeAsImage(pageToConvertAsImage).then((resolve) => {
  console.log("Page 1 is now converted as image");

  return resolve;
});
```
This is how it's supposed to be used. 

The `.convert()` function is now part of the variable you assign fromBuffer/fromPath to. In this case, is `storeAsImage` is used as a function and will work the same as calling `.convert`.

**Comment by hadrien-f** [comment]
#94 I tried to update the readme then

**Comment by yakovmeister** [comment]
merged #94 https://github.com/yakovmeister/pdf2image/pull/94
